sdk/daml-lf/ledger-api-value/src/main/protobuf/com/daml/ledger/api/v2/value.proto

```diff
@@ -142,9 +134,24 @@ message Identifier {
   string entity_name = 3;
 
   // Name of the package.
-  // This is always provided as reading from the Ledger API.
-  // This is not required as provided on Ledger API commands/requests.
-  // Optional
+  //
+  // This is always set on responses from the Ledger API.
+  //
+  // This can be set in a Ledger API request to select a specific package name
+  // *provided* the request allows for package-name selection.
```

Comment:
Do we have such where we disallow, I wonder?

Reply:
We may have in the future.

Interfaces are currently sort of referred to by package-id, as each package-ids interface is considered different. That might change though.

I'd just keep the comment. It will age better.